### DISCUSSION TITLE
Can anyone give me a few social networking ios mobile apps that are using FastAPI as the backend framework?

### DISCUSSION THREAD

**Discussion by lanxu2000** [fastapi/fastapi#11431] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
I don't have any code to share at this point. Just a general question hoping to get an answer to decide my next step. I do like what FastAPI offers. But the lack of user cases makes me a little concerned. Thank you for sharing your insights!
```

### Description

I'm unable to find user cases where FastAPI is deployed as the backend for a social networking app through online search. 

### Operating System

Linux, macOS

### FastAPI Version

I haven't decided whether to use FastAPI for the app yet. Still exploring

### Pydantic Version

Still exploring

### Python Version

Still exploring

**Comment by acidjunk** [comment, marked as answer]
FastAPI is perfect for rest and (combined with strawberry) graphql api's.

So you can probably build what you want with it: and scale it.

Not sure what the functional requirements are; based on your description. Are you aiming for world domination?

**Reply by lanxu2000** [reply]
Thank you for the response, Rene! 

No, conquering the world is not my ambition :-) I just want to build a simple functional social app that offers a good user experience to whoever chooses to sign up to my app. 

FastAPI has been around for a few years. It seems solid and strong. I like what I've seen and learned so far. But the fact that I can't find a good social app name out there just puzzles me a bit. Hope people with more experience can share some thoughts or insights as to why.
